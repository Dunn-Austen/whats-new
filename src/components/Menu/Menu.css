.genre-text {
  background-color: #c9e1ff;
}

.btn-container {
  background-color: #c9e1ff;
}

.menu-btn {
  background-color: white;
  border: 1px solid #1B009F;
  height: 50px;
  width: 100px;
}

.menu-btn:hover {
  background-color: #ccafbd;
  color: black;
}
