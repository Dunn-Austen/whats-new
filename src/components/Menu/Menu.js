import React from 'react';
import './Menu.css'

const Menu = (props) => {


  return (
    <nav className="menu">
      <button>Entertainment</button>
      <button>Health</button>
      <button>Local</button>
      <button>Science</button>
      <button>Technology</button>
    </nav>
  )
}

export default Menu;
