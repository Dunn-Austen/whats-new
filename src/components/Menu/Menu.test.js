import React from 'react';
import Menu from './Menu';
import { shallow } from 'enzyme';

describe('Menu', () => {

  it('should match the snapshot with all data correctly passed', () => {
    const wrapper = shallow(<Menu
              entertainment={[{id: 1, headline: 'A' , img: 'B' , description: 'C', url: 'D'}]}
              health={[{id: 2, headline: 'A' , img: 'B' , description: 'C', url: 'D'}]}
              local={[{id: 3, headline: 'A' , img: 'B' , description: 'C', url: 'D'}]}
              science={[{id: 4, headline: 'A' , img: 'B' , description: 'C', url: 'D'}]}
              switchStateGenre={jest.fn()}
              technology={[{id: 5, headline: 'A' , img: 'B' , description: 'C', url: 'D'}]}
            />);

    expect(wrapper).toMatchSnapshot();
  })
})
