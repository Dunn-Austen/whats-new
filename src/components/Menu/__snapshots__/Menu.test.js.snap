// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot with all data correctly passed 1`] = `
<nav
  className="menu"
>
  <button
    className="menu-btn"
    id="entertainment-btn"
    onClick={[Function]}
  >
    Entertainment
  </button>
  <button
    className="menu-btn"
    id="health-btn"
    onClick={[Function]}
  >
    Health
  </button>
  <button
    className="menu-btn"
    id="local-btn"
    onClick={[Function]}
  >
    Local
  </button>
  <button
    className="menu-btn"
    id="science-btn"
    onClick={[Function]}
  >
    Science
  </button>
  <button
    className="menu-btn"
    id="technology-btn"
    onClick={[Function]}
  >
    Technology
  </button>
</nav>
`;
